### ISSUE TITLE
Few issues

### ISSUE THREAD

**Issue by kantlivelong** [jonfairbanks/OctoPrint-NetworkHealth#8]
Hello,

(This was made as of bee1bd9313135edc03a506fad30112a67888b15a)

Just was poking at this plugin and wanted give you a heads up on a few things:
- Your current release version **1.0.0** doesn't run on OctoPrint and will never receive updates. You might want to push 1.0.1 or retag 1.0.0 to a newer working commit
- While not explicitly required you should move [these](https://github.com/jonfairbanks/OctoPrint-NetworkHealth/blob/bee1bd9313135edc03a506fad30112a67888b15a/octoprint_NetworkHealth/__init__.py#L43-L59) into the NetworkHealthPlugin class.
- Finally, while it may not be logged; changes made via the `ip` command require root privileges. So as it stands when those commands are run they do not actually do anything. In order for this to work the plugin would need elevated privileges via sudo. By default on OctoPi the pi user does not have sudo rights to the `ip` command so the user would need to manually configure that. Take a look at `/etc/sudoers` and `/etc/sudoers.d` Example ip command error below:
```
pi@octopi:~ $ ip link set eth0 down
RTNETLINK answers: Operation not permitted
```

**Comment by jonfairbanks** [comment]
Thank you for this feedback. I will issue a fix for these shortly.

**Comment by jonfairbanks** [comment]
@kantlivelong I have released 1.0.1 which should take care of some of this feedback.

https://github.com/jonfairbanks/OctoPrint-NetworkHealth/releases/tag/1.0.1

Please let me know if you are seeing otherwise or have other suggestions. 

Thanks!
